src/ui/components/accessibility/BasicAccessibilityInfo.native.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+
+import { render, screen } from 'tests/utils'
+import { BasicAccessibilityInfo } from 'ui/components/accessibility/BasicAccessibilityInfo'
+
+describe('BasicAccessibilityInfo', () => {
+  it('renders all handicap information', () => {
```

Comment:
dans tous les autres fichiers de test on utilise plutôt la syntaxe "should render", on pourrait l'utiliser aussi ici ! (C'est optionnel)

Reply:
J'avoue j'ai juste changé le nom du fichier et adapté mais tu as raison !